### ISSUE TITLE
Custom snackbar example source imports incorrect

### ISSUE THREAD

**Issue by gknapp** [iamhosseindhv/notistack#337]
The [custom snackbar example](https://iamhosseindhv.com/notistack/demos#custom-snackbar) in the docs show `SnackbarContent` being imported from notistack but there is no export member for this in notistack (1.0.1).

```
import { makeStyles } from '@material-ui/core/styles';
import { useSnackbar, SnackbarContent } from 'notistack';
```

![image](https://user-images.githubusercontent.com/143027/101014094-373e6a00-355d-11eb-932f-e839b9a1dcf2.png)

## Expected Behavior
Should the docs show `SnackbarContent` being imported from `@material-ui/core`? Or should the docs import be `SnackbarContentCallback` from notistack?

```
import { makeStyles } from '@material-ui/core/styles';
import { SnackbarContent } from '@material-ui/core';
import { useSnackbar } from 'notistack';
```

## Current Behavior
JS / TS / VSCode complains the import member is not found.

## Context
Building a custom snackbar

## Your Environment

| Tech         | Version |
|--------------|---------|
| Notistack | v1.0.1 |
| React        | v16.13.1 |
| Browser      | n/a |

**Comment by Bopzor** [comment]
Related issue #311

**Comment by iamhosseindhv** [comment]
Example source code is correct. `SnackbarContent` should be imported from `notistack`.

Snackbar Content: https://github.com/iamhosseindhv/notistack/tree/master/src/SnackbarContent
Exported from notistack: https://github.com/iamhosseindhv/notistack/blob/master/src/index.js#L2

**Comment by gknapp** [comment]
[This comment](https://github.com/iamhosseindhv/notistack/issues/311#issuecomment-751915301) in #311 suggests the export might not be working as intended?

**Comment by gknapp** [comment]
Fixed in #343
